spec/mailers/demo_mailer_spec.rb

```diff
@@ -34,4 +34,35 @@
       )
     end
   end
+
+  context 'invitation mailer' do
+    let(:work_group) { create(:work_group) }
+    let(:student_work_group) { create(:student_work_group, work_group: work_group) }
+    let(:worksheet) { create(:worksheet, work_group: work_group) }
+    let(:user_email) { student_work_group.student.email }
+    let(:image_url) { worksheet.image_url }
+    subject do
+      DemoMailer.with(
+        students: user_email,
+        student_group: work_group,
+        image_url: image_url
+      ).send_worksheets
+    end
+
+    it 'renders the subject' do
+      expect(subject.subject).to eql("[Odaiba: #{work_group.name} in #{work_group.classroom.name}] Successfully submitted worksheet")
+    end
+
+    it 'renders the receiver email' do
+      expect(subject.to).to eql([user_email])
+    end
+
+    it 'renders the sender email' do
+      expect(subject.from).to eql(['infoodaiba@gmail.com'])
+    end
+
+    it 'renders the image url' do
+      # expect(subject.body.encoded).to match('')
```

Comment:
It runs fine, but I'm not sure if this is truly returning the image url?

Reply:
the worksheet factory (what you are using in line 41) creates a worksheet without an image_url
in order to test that this really works, we need to add the image url to the worksheet and then check here that it's really rendered

Reply:
Okay, I will re-use the image url used for the test worksheet and check!